packages/expo-av/ios/EXAV/EXAV.m

```diff
@@ -387,7 +387,7 @@ - (void)_runBlock:(void (^)(EXAVPlayerData *data))block
   if (data) {
     block(data);
   } else {
-    reject(@"E_AUDIO_NOPLAYER", nil, UMErrorWithMessage(@"Player does not exist."));
+    reject(@"E_AUDIO_NOPLAYER", @"Player does not exist.", nil);
```

Comment:
I'd like this error to give at least some clue what should be done in order to prevent it 🤔 
I imagine sth like this:
```suggestion
    reject(@"E_AUDIO_NOPLAYER", @"Player does not exist. Ensure player component is rendered.", nil);
```
Not sure about actual reason for this error though 😅